Documentation/op-guide/configuration.md

```diff
@@ -1,6 +1,10 @@
 # Configuration flags
 
-etcd is configurable through command-line flags and environment variables. Options set on the command line take precedence over those from the environment.
+etcd is configurable through a configuration file, various command-line flags, and environment variables.
+
+A reusable configuration file is a YAML file made of one or more command-line flags described below. In order to use this file, specify the file path as a value to the `--config-file` flag. The [sample configuration file][sample-config-file] can be used as a starting point to create a new configuration file as needed.
+
+Options set on the command line take precedence over those from the environment. If a configuration file is provided, other command line flags will be ignored.
```

Comment:
If a configuration file is provided, will env also be ignored? or they will be overwritten?

Reply:
@xiang90 env variables will be ignored in that case.

Reply:
can we clarify that here too besides `other command line flags will be ignored`?

Reply:
Sure

Reply:
probably we should also add some logging in etcd itself saying `config file is provided, xxx are ignored.`

Reply:
no need to be in this PR though.

Reply:
Agree, both suggestions makes sense!

Reply:
I have created second commit with your suggestions related to ignoring flags. Thanks!